app/controllers/admin/cases_controller.rb

```diff
@@ -145,6 +158,18 @@ def create_sar_params
     ).merge(type: "Case::SAR")
   end
 
+  def create_ico_params
```

Comment:
My previous comment about removing `create_params` was also for `create_{foi,ico,sar}_params` ... I don't think any of these are used, so let's get rid of them all!!! BURN PARAMS< BURN!!